From c1d060f8c19239412af0cfa51414db2435b6782b Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 27 Feb 2015 15:10:29 +0100
Subject: [ABRT PATCH] ureport: rename the event to submit_uReport

Because we don't want to show the event in report-gtk in the list of available
reporters (report-gtk includes all events prefixed with 'report' in that list).

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/daemon/abrt.conf          | 4 ++--
 src/lib/abrt_conf.c           | 2 +-
 src/plugins/ccpp_event.conf   | 2 +-
 src/plugins/koops_event.conf  | 2 +-
 src/plugins/python_event.conf | 2 +-
 src/plugins/vmcore_event.conf | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/daemon/abrt.conf b/src/daemon/abrt.conf
index 6cc3376..0766856 100644
--- a/src/daemon/abrt.conf
+++ b/src/daemon/abrt.conf
@@ -28,9 +28,9 @@ DeleteUploaded = no
 # In order to run this event automatically after detection, the
 # AutoreportingEnabled option must be configured to 'yes'
 #
-# Default value: report_uReport
+# Default value: submit_uReport
 #
-AutoreportingEvent = report_uReport
+AutoreportingEvent = submit_uReport
 
 # Enables automatic running of the event configured in AutoreportingEvent option.
 #
diff --git a/src/lib/abrt_conf.c b/src/lib/abrt_conf.c
index f611b0c..4f23535 100644
--- a/src/lib/abrt_conf.c
+++ b/src/lib/abrt_conf.c
@@ -90,7 +90,7 @@ static void ParseCommon(map_string_t *settings, const char *conf_filename)
         g_hash_table_remove(settings, "AutoreportingEvent");
     }
     else
-        g_settings_autoreporting_event = xstrdup("report_uReport");
+        g_settings_autoreporting_event = xstrdup("submit_uReport");
 
     GHashTableIter iter;
     const char *name;
diff --git a/src/plugins/ccpp_event.conf b/src/plugins/ccpp_event.conf
index 07d2224..5da99c8 100644
--- a/src/plugins/ccpp_event.conf
+++ b/src/plugins/ccpp_event.conf
@@ -37,7 +37,7 @@ EVENT=analyze_LocalGDB analyzer=CCpp
         abrt-action-analyze-ccpp-local --without-di --without-bz --without-bodhi
 
 # Send micro report
-EVENT=report_uReport analyzer=CCpp
+EVENT=submit_uReport analyzer=CCpp
         /usr/libexec/abrt-action-ureport
 
 # update ABRT database after successful report to bugzilla
diff --git a/src/plugins/koops_event.conf b/src/plugins/koops_event.conf
index bed4c80..6f6c1ef 100644
--- a/src/plugins/koops_event.conf
+++ b/src/plugins/koops_event.conf
@@ -13,7 +13,7 @@ EVENT=post-create analyzer=Kerneloops
 # report
 
 # Send micro report
-EVENT=report_uReport analyzer=Kerneloops
+EVENT=submit_uReport analyzer=Kerneloops
         /usr/libexec/abrt-action-ureport
 
 # update ABRT database after successful report to bugzilla
diff --git a/src/plugins/python_event.conf b/src/plugins/python_event.conf
index d498353..deae396 100644
--- a/src/plugins/python_event.conf
+++ b/src/plugins/python_event.conf
@@ -2,7 +2,7 @@ EVENT=post-create analyzer=Python
         abrt-action-analyze-python
 
 # Send micro report
-EVENT=report_uReport analyzer=Python
+EVENT=submit_uReport analyzer=Python
         /usr/libexec/abrt-action-ureport
 
 # update ABRT database after successful report to bugzilla
diff --git a/src/plugins/vmcore_event.conf b/src/plugins/vmcore_event.conf
index 1ce77f1..92300e3 100644
--- a/src/plugins/vmcore_event.conf
+++ b/src/plugins/vmcore_event.conf
@@ -11,7 +11,7 @@ EVENT=analyze_VMcore analyzer=vmcore
 
 
 # Send micro report
-EVENT=report_uReport analyzer=vmcore
+EVENT=submit_uReport analyzer=vmcore
         /usr/libexec/abrt-action-ureport
 
 # update ABRT database after successful report to bugzilla
-- 
1.8.3.1

